### PR TITLE
WIP: Add coding conventions to code.

### PR DESCRIPTION
**Pull request by jaanjah**
This merge request will add coding conventions to the code, reason behind this pull request is to improve code quality and make a consistent codebase.

Somewhat related to: https://github.com/PulsarcGame/Pulsarc/issues/37

If you disagree with any of my changes feel free to make a review why this is a bad idea.

### CODE REVIEW

**Review by Adrriii — CHANGES_REQUESTED** [review]
The IniData attribute is named "get" because it looks like Config.get[category][element]. It is easier to remember and makes sense.

I only set attributes/methods private for preventing accidental use which could harm the program. keyType doesn't really need to be private since some day for optimization we could use it to get the Keys type (which can't be obtained by another technique but would still remain constant).

The rest looks good, thanks !

File: Pulsarc/Utils/KeyVal.cs [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
         public Key_ Key { get; set; }
         public Value_ Value { get; set; }
-
         public KeyVal() { }
 
         public KeyVal(Key_ key, Value_ val)
         {
-            Key = key;
-            Value = val;
+            this.Key = key;
```
Comment (Adrriii) [inline Pulsarc/Utils/KeyVal.cs]: Why add unnecessary this ?

Comment (jaanjah) [reply Pulsarc/Utils/KeyVal.cs]: To use those values in that instance, those are indeed unnecessary if you plan not adding anything else in that class that uses those variables.

EDIT: I guess for the current usage I can remove `this` keyword, sorry.

Comment (Adrriii) [reply Pulsarc/Utils/KeyVal.cs]: This object is just a workaround since KeyValuePair is read-only, and not this one

Comment (jaanjah) [reply Pulsarc/Utils/KeyVal.cs]: Changed it back to how it was before.

**Pull request closed without merging**